client/src/components/new-request/search.vue

```diff
@@ -190,44 +273,59 @@ export default class NewSearch extends Vue {
   get locationOptions () {
     return newReqModule.locationOptions
   }
+  get locationText () {
+    return newReqModule.locationText
+  }
   get nameIsEnglish () {
-    return newReqModule.nameIsEnglish
+    return !newReqModule.nameIsEnglish
```

Comment:
This doesn't make sense 😆

Reply:
I should refactor the name 😆 
The verbiage update here called for the opposite of what it currently was, functionally as well. This was the easy path, but i'll clear this up.

Reply:
Gotcha. Thanks, please rename, lol.

Reply:
Found an alternative to renaming anything at all!

![Screen Shot 2020-11-19 at 10 33 01 AM](https://user-images.githubusercontent.com/53542131/99708507-bf4b4c80-2a52-11eb-8fb3-a093fdc3ceee.png)

Reply:
Essentially just flip flops the state vs value, so the property name and all instances of it still make sense.

Reply:
Yikes, but OK.

Reply:
hahah it's not that scary!